python.md

```diff
@@ -1233,6 +1234,18 @@ Related reading:
 
 - http://stackoverflow.com/questions/19074745/python-docstrings-descriptions-vs-comments
 
+### <a name="spelling">Spelling</a>
+
+Consistent spelling is helpful because it decreases the risk of incorrect variable references. Where applicable, follow the spelling used in external libraries. For example, the Django Rest Framework uses the American English spelling for `serializer` which is therefore prefered. Elsewhere aim for conformity with the pre-existing choices. For example, given the following split:
+
+```bash
+> rg organize | wc -l
+> 10
+> rg organise | wc -l
+> 86
+```
+
+Choose `organise`. Finding and renaming all instances of `organize` would earn you maximum points.
```

Comment:
I thought we once had a discussion on Slack where we agreed to use American spellings of everything 😕I believe @sandwichsudo asked it? It was a long time ago 😛 For the record - I don't have a strong opinion of this; I'd be happy to stick to whatever we choose.

Reply:
I don't have strong opinions either and I'm not super convinced by this PR. It feels a little bit like a non-problem so not of huge consequence whatever we decide (one of the main benefits of writing something up might just be to stop the question resurfacing). It could be overly prescriptive to decide on a language choice (we don't want to waste energy arguing about the Queen's english in code reviews) and in practice maintaining American English, because the majority of us aren't in the habit of writing it, could be quite hard. We'd like to follow the libraries we use (which I'm guessing is a mix of both types of spelling) but we also want as much consistency as possible. It's a tough one. 🤷‍♂️

Reply:
I would drop this part about "organise". It's not relevant to include codebase search results in a public repo like this.